src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java

```diff
@@ -0,0 +1,52 @@
+// to stop adding it:
+// git reset src/main/java/com/spartronics4915/frc2020/commands/Spin4TimesCommand.java
+package com.spartronics4915.frc2020.commands;
+
+import com.spartronics4915.frc2020.subsystems.PanelRotator;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Spin4TimesCommand extends CommandBase {
+
+    private PanelRotator mPanelRotator;
+    public double spins;
+    public String firstColor;
+    public String lastColor;
+    public String currentColor;
+    // You should only use one subsystem per command. If multiple are needed, use a CommandGroup.
+    public Spin4TimesCommand(PanelRotator mSubsystem) {
+      mPanelRotator = mSubsystem;
```

Comment:
addRequirements

Reply:
@j-james what are you saying? I don't comprehend.

Reply:
Inside of your constructor, you need to tell the CommandScheduler what subsystem your Command requires, so that it doesn't schedule two for the same subsystem. In this case, you'd call `addRequirements(mPanelRotator)`.